quark.c

```diff
@@ -4844,6 +4825,8 @@ quark_queue_get_event(struct quark_queue *qq)
 			break;
 		}
 
+		if (qev)
+			qev->time = quark.boottime + raw->time;
```

Comment:
```suggestion
		if (qev != NULL)
			qev->time = quark.boottime + raw->time;
```
To be consistent with the rest of the code